api/capi/include/nntrainer.h

```diff
@@ -41,29 +41,39 @@ extern "C" {
  * @brief A handle of an NNTrainer model.
  * @since_tizen 6.x
  */
-typedef void *ml_nnmodel_h;
+typedef void *ml_train_model_h;
 
 /**
  * @brief A handle of an NNTrainer layer.
  * @since_tizen 6.x
  */
-typedef void *ml_nnlayer_h;
+typedef void *ml_train_layer_h;
 
 /**
  * @brief A handle of an NNTrainer optimizer.
  * @since_tizen 6.x
  */
-typedef void *ml_nnopt_h;
+typedef void *ml_train_optimizer_h;
 
 /**
  * @brief Enumeration for the neural network layer type of NNTrainer.
  * @since_tizen 6.x
  */
 typedef enum {
-  ML_LAYER_TYPE_INPUT = 0, /**< Input Layer */
-  ML_LAYER_TYPE_FC,        /**< Fully Connected Layer */
-  ML_LAYER_TYPE_UNKNOWN    /**< Unknown Lyaer */
-} ml_layer_type_e;
+  ML_TRAIN_LAYER_TYPE_INPUT = 0, /**< Input Layer */
+  ML_TRAIN_LAYER_TYPE_FC,        /**< Fully Connected Layer */
+  ML_TRAIN_LAYER_TYPE_UNKNOWN    /**< Unknown Layer */
```

Comment:
Are you going to leave this UNKNOWN as it is?

Reply:
Yes keeping the unknown. This is for the later use cases when getting layer and its info will be allowed.

Reply:
BTW, how about making unknown sth like 9999 or -1 considering that we are adding layer types

Reply:
Added for both layer and optimizer unknown type.